### PR TITLE
Cloudportal release CP237

### PR DESCRIPTION
**Pull request by mbyregowMx**
release notes updated for CP237 release

### CODE REVIEW

File: content/releasenotes/developer-portal/mendix-cloud.md [hunk 1]
```diff
@@ -10,6 +10,14 @@ These release notes cover changes to deployment to [Mendix Cloud](/developerport
 
 For information on the current status of deployment to Mendix Cloud and any planned releases see [Mendix Status](https://status.mendix.com/).
 
+## 2022
+
+### January 6th, 2022
+
+#### Fixes
+
+* We fixed an issue where start environment deploy API failed to perform the action. (Tickets 132011)
```
Comment (MarkvanMents) [inline content/releasenotes/developer-portal/mendix-cloud.md]: @mbyregowMx 
Hi Madhu,
According to the ticket, this was resolved on 11 November last year? I'm a bit puzzled as to why we should be mentioning a fix now?

File: content/releasenotes/developer-portal/mendix-cloud.md [hunk 2]
```diff
@@ -10,6 +10,14 @@ These release notes cover changes to deployment to [Mendix Cloud](/developerport
 
 For information on the current status of deployment to Mendix Cloud and any planned releases see [Mendix Status](https://status.mendix.com/).
 
+## 2022
+
+### January 6th, 2022
+
+#### Fixes
+
+* We fixed an issue where the start environment call to the deploy API failed to start the environment. (Ticket 132011)
```
Comment (MarkvanMents) [inline content/releasenotes/developer-portal/mendix-cloud.md:19]: @mbyregowMx
Hi Madhu,
According to the ticket, this was resolved on 11 November last year? I'm a bit puzzled as to why we should be mentioning a fix now?

**Review by MarkvanMents — APPROVED** [review]
OK to publish, but have asked a question. Not a blocker to publication.

**Comment by mbyregowMx** [comment]
@MarkvanMents The issue was resolved temporarily, now a permanent fix has been made by updating the underlying implementation. This was committed as part of the RCA for this ticket.

File: content/releasenotes/developer-portal/mendix-cloud.md (hunk 2 above)
Comment (MarkvanMents) [reply content/releasenotes/developer-portal/mendix-cloud.md:19]: OK - this is now a permanent fix rather than a workaround.

**Comment by MarkvanMents** [comment]
Not released - will be informed when it is released.

**Comment by MarkvanMents** [comment]
Released - information via Slack.

**Pull request merged**